### PR TITLE
update owner files in samples and test

### PR DESCRIPTION
**Pull request by gaoning777**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/1785)

### CODE REVIEW

**Comment by numerology** [comment]
Thanks! @gaoning777

File: test/OWNERS [hunk 1]
```diff
@@ -1,10 +1,6 @@
 approvers:
-  - vicaire
-  - IronPan
```
Comment (IronPan) [inline test/OWNERS]: why removing me? :)

Comment (gaoning777) [reply test/OWNERS]: You are already the owner on the top level, which naturally means you are the approver for everything. If you want to keep your name here, I don't have problems with that though.

Comment (IronPan) [reply test/OWNERS]: this would help people triage the issue at finer level. could you add it back?

Comment (gaoning777) [reply test/OWNERS]: Done.

Comment (IronPan) [reply test/OWNERS]: can u add me back to the approver?

Comment (gaoning777) [reply test/OWNERS]: ?? You are still in the approver list. 
Can you check again? Thanks

Comment (IronPan) [reply test/OWNERS]: nvm order changed i crossed my eyes.

**Pull request merged**